observations/obs_converters/Ameriflux/fluxnetfull_to_obs.f90

```diff
@@ -0,0 +1,1180 @@
+
+program Fluxnetfull_to_obs
+
+!------------------------------------------------------------------------
+!
+!   Fluxnetfull_to_obs - a program that converts Ameriflux/Fluxnet FULLSET eddy
+!                        covariance tower data of NEE, GPP, RE, latent heat and sensible
+!                        heat fluxes into DART obs_seq formatted files. Works on native time 
+!                        resolution (HH, HR) that ED tower data or collected or at coarser
+!                        aggregated time resolution files (DD,WW,MM) based on ONEFLUX
+!                        methodology (Pastorello et al., 2020)
+    
+
+use         types_mod, only : r8, MISSING_R8
+
+use     utilities_mod, only : initialize_utilities, finalize_utilities, &
+                              register_module, error_handler, E_MSG, E_ERR, &
+                              open_file, close_file, do_nml_file, do_nml_term, &
+                              check_namelist_read, find_namelist_in_file, &
+                              nmlfileunit, logfileunit
+
+use  time_manager_mod, only : time_type, set_calendar_type, GREGORIAN, &
+                              set_date, set_time, get_time, print_time, &
+                              print_date, operator(-), operator(+), operator(>), &
+                              operator(<), operator(==), operator(<=), operator(>=), &
+                              operator(/)
+
+use      location_mod, only : VERTISHEIGHT
+
+use  obs_sequence_mod, only : obs_sequence_type, obs_type, read_obs_seq, &
+                              static_init_obs_sequence, init_obs, write_obs_seq, &
+                              init_obs_sequence, get_num_obs, &
+                              set_copy_meta_data, set_qc_meta_data
+
+use      obs_kind_mod, only : TOWER_SENSIBLE_HEAT_FLUX, &
+                              TOWER_NETC_ECO_EXCHANGE,  &
+                              TOWER_LATENT_HEAT_FLUX,   &
+                              TOWER_ER_FLUX,            &
+                              TOWER_GPP_FLUX
+
+implicit none
+
+character(len=*), parameter :: source   = 'Fluxnetfull_to_obs.f90'
+
+!-----------------------------------------------------------------------
+! Namelist with default values
+!-----------------------------------------------------------------------
+
+character(len=256) :: text_input_file = 'textdata.input'
+character(len=256) :: obs_out_file    = 'obs_seq.out'
+real(r8)           :: timezoneoffset  = -1.0_r8
+real(r8)           :: latitude        = -1.0_r8
+real(r8)           :: longitude       = -1.0_r8
+real(r8)           :: elevation       = -1.0_r8
+real(r8)           :: flux_height     = -1.0_r8
+! A maxgooqc=3 allows for good=1, medium=2, and poor=3 quality gap-filled data
+real(r8)           :: maxgoodqc       = 3.0_r8
+! Always true except for latent,sensible heat and NEE for hourly time periods
+! Can only be false of HH or HR time resolution
+logical            :: gap_filled      = .true.
```

Comment:
Lines 61 and 62 mean that for the coarse time resolution (DD, WW, MM), gap_filled for NEE, latent, sensible heat, GPP and RE is true, and for the native time resolution (HH, HR), gap_filled for NEE, latent, sensible heat, GPP and RE can be true or false, correct?